ScreenManager.c

```diff
@@ -147,7 +148,7 @@ static void checkRecalculation(ScreenManager* this, double* oldTime, int* sortTi
       // always update header, especially to avoid gaps in graph meters
       Header_updateData(this->header);
       // force redraw if the number of UID digits was changed
-      if (Process_uidDigits != oldUidDigits) {
+      if (Process_uidDigits != oldUidDigits || Process_pidDigits != oldPidDigits) {
          *force_redraw = true;
       }
```

Comment:
Just make this two blocks.
```suggestion
      if (Process_uidDigits != oldUidDigits) {
         *force_redraw = true;
      }
      if (Process_pidDigits != oldPidDigits) {
         *force_redraw = true;
      }
```

Also, that comment above needs updating.

Reply:
I fixed the comment above.

I reverted this into one conditional to take advantage of the short-circuit logic in the evaluation. Likely doesn't make a huge difference in performance though.